cmd/neofs-node/config.go

```diff
@@ -93,6 +94,13 @@ const (
 
 	cfgReBootstrapEnabled  = "bootstrap.periodic.enabled"
 	cfgReBootstrapInterval = "bootstrap.periodic.interval"
+
+	cfgObjectPutPoolSize       = "pool.object.put"
```

Comment:
I suggest to underline `size` in config path.

Reply:
> I suggest to underline `size` in config path.

How about `pool.size.object.put`?

Reply:
Seems good. Alternative: `pool.object.put.size`.

Reply:
Fixed.